Open auction contract and testing (#1)
SeherSaylik/open-auction: README.md

# OpenAuction

> A summary of the project.

Reviewer: I updated the PR with some changes. Check my last commit and update the summary and "How it Works?" on this file.

# OpenAuction

## What is this project for?

In this project it is aimed to create an open auction without any time limit. The person who deploys the contract will be the contract owner. And it has the right to start and stop an auction at any time. 

The contract simply ranks each new bid and always retains only the highest. When the owner of the contract calls the 'endAuction' function to end the auction, the address with the highest bid wins.

## Install necessary packages

`npm install`

## In order to start decentralized auction application 

`npm run start`

**Steps need to do**

> 1. Firstly compile the contract (hardhat compile) and get artifacts directory. We will need ABI(Application Binary Interface) file inside of it.

Reviewer: ```suggestion
1. Firstly compile the contract with `hardhat compile` and get the artifacts directory. We will have an ABI(Application Binary Interface) file inside of it.
```
